site/faq.md

```diff
@@ -91,6 +91,14 @@ You must permit traffic to flow through TCP 6783 and UDP 6783/6784,
 which are Weave’s control and data ports.
 
 
+**<a name=own-image></a>Q: Why do you use your own Docker image `weaveworks/ubuntu` ? **
```

Comment:
The space after the question mark is causing the formatting to go awry. Should be like this: 

**<a name=own-image></a>Q: Why do you use your own Docker image `weaveworks/ubuntu`?**